pages/index.vue

```diff
@@ -26,6 +28,7 @@ export default Vue.extend({
     LandingDaoProposals,
     // LandingJobOffers,
     LandingLatestProjects,
+    LandingPearlxDao
```

Comment:
nitpick (non-blocking): I am not so sure about the "x". I think we can just drop it completely. PearlDao looks good to me.